apps/nextjs-website/react-components/components/HeroChips/HeroChips.helpers.tsx

```diff
@@ -0,0 +1,56 @@
+import { Stack, Chip } from '@mui/material';
+import { ChipProps } from '@react-components/types/HeroChips/HeroChips.types';
+
+export const ChipsBlock = ({
+  chips,
+  theme,
+}: {
+  chips: ReadonlyArray<ChipProps>;
+  theme: 'light' | 'dark';
+}) => {
+  const handleChipClick = (targetID: string) => {
+    const element = document.getElementById(targetID);
+    if (element) {
+      element.scrollIntoView({ behavior: 'smooth' });
+    }
+  };
+
+  return (
+    <Stack
+      direction='row'
+      spacing={1}
+      mt={2}
+      sx={{
+        width: '100%',
+        justifyContent: 'center',
+      }}
+    >
+      <Stack
+        direction='row'
+        spacing={1}
+        sx={{
+          maxWidth: '600px',
+          flexWrap: 'wrap',
+          rowGap: '8px',
+          justifyContent: 'center',
+        }}
+      >
+        {chips.map((chip, index) => (
+          <Chip
+            key={index}
+            label={chip.label}
+            onClick={() => handleChipClick(chip.targetID)}
```

Comment:
```suggestion
            href=`#${chip.targetID}`
```
This needs testing, but an internal link to the section should suffice.